docs/docs/eslint.md

```diff
@@ -31,59 +29,141 @@ rm .prettierrc
 touch .eslintrc.js
 ```
 
-### Configuring ESLint
+## Configuring ESLint
 
-We recommend copying our default .eslintrc.js content below to your newly created `.eslintrc.js` file and modifying it per your needs. Reference ESLint's [rules documentation](https://eslint.org/docs/rules/) for more options.
+To get started configuring ESLint we recommend copying our default ESLint content below to your newly created `.eslintrc.js` file and modifying it per your needs. Reference ESLint's [rules documentation](https://eslint.org/docs/rules/) for more options. Sample rules from the plugins are also provided to demonstrate their capabilities.
```

Comment:
These edits make a lot of sense to me! I do think it would help to explain which packages you're installing, though. For example, I don't see a mention of what `jsx-a11y` is (even though I personally already know). What do these particular packages do for you?